modules/simple-payments/simple-payments.css

```diff
@@ -0,0 +1,21 @@
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message {
+	display: none;
+}
+
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message,
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message a {
+	color: #fff;
+	padding: 1em;
+}
+
+body .jetpack-simple-payments__wrapper div.jetpack-simple-payments__purchase-message.show {
```

Comment:
We don't need to add specificities to others. We only need the bloated selector for the `<p>`.

```
body .jetpack-simple-payments__wrapper p.jetpack-simple-payments__purchase-message {
   color: #fff;
}
```
The rest shouldn't just be fine as they were.

Are we displaying links in the message?